### PR TITLE
fix : Warning: Undefined array key other_linked_objects in /home/http…

### PR DESCRIPTION
**Pull request by grandoc**
…d/vhosts/aflac.fr/domains/dev.aflac.fr/httpdocs/compta/facture/card.php on line 1470

### CODE REVIEW

File: htdocs/compta/facture/card-rec.php [hunk 1]
```diff
@@ -1186,6 +1187,7 @@
 		}
 
 		// Call Hook formConfirm
+		$formconfirm = '';
```
Comment (eldy) [inline htdocs/compta/facture/card-rec.php:1190]: You erase the formconfirm that is set few lines before

**Comment by grandoc** [comment]
But it was commented, and so make an error

File: htdocs/compta/facture/card-rec.php (hunk 1 above)
Comment (grandoc) [reply htdocs/compta/facture/card-rec.php:1190]: I have fixed it in an other pull request

Comment (eldy) [reply htdocs/compta/facture/card-rec.php:1190]: So i close this PR

**Pull request closed without merging**